tests/test_outputs.py

```diff
@@ -47,12 +49,66 @@ def test_logging(temp_dir):
     assert b'test log message' in logs
 
 
-def test_collect_file(temp_dir):
+@pytest.fixture
+def test_file(temp_dir):
     test_file = os.path.join(temp_dir, 'test_file.txt')
 
     with open(test_file, 'w') as f:
-        f.write('test content')
+        f.write('MZtest content')
+
+    yield test_file
+
+
+@pytest.fixture
+def test_pe_file():
+    DATA_ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), 'data'))
+
+    return OSFileSystem('/').get_fullpath(os.path.join(DATA_ROOT, 'MSVCR71.dll'))
 
+
+def test_collect_file_info(temp_dir, test_file):
+    output = Outputs(temp_dir, None, False)
+    output.add_collected_file_info('TestArtifact', OSFileSystem('/').get_fullpath(test_file))
+    output.close()
+
+    with Reader(output_file_content(temp_dir, '*-file_info.jsonl').splitlines()) as jsonl:
+        record = jsonl.read()
+
+        assert '@timestamp' in record
+        assert record['file']['path'].endswith('test_file.txt')
+        assert record['file']['size'] == 14
+        assert record['file']['mime_type'] == "application/x-msdownload"
+        assert record['file']['hash']['md5'] == "10dbf3e392abcc57f8fae061c7c0aeec"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/fastir-build/fastir_artifacts/pullRequest?prid=5744135)